Automattic/woocommerce-payments
# Support tokenized cart PRBs on blocks cart and checkout

Fixes #8527 

#### Changes proposed in this Pull Request

This adds tokenized cart PRBs to the cart and checkout blocks. The goal is implement the PRB component on the blocks cart and checkout pages, removing any reliance on the older implementation.









#### Testing instructions





* Disable the Express Checkout Buttons feature flag as it's now the default - `npm run wp option update _wcpay_feature_stripe_ece 0`. To verify it's disabled you can check the WooPayments Express Checkout settings for a border radius option. Border radius only appears for the ECE buttons.
* Enable the tokenized cart PRB feature flag in the dev tools
* Visit the blocks cart page and check that the PRB is appearing
* Click the PRB and check that totals are correct
* Update shipping methods and options and check that totals update correctly
* Place an order and verify that the order totals are correct
* Check that the **Origin** in the order is no `Unknown`. 
* Repeat the above 4 steps on the checkout block

-------------------


**Post merge**
